processing/src/main/java/org/apache/druid/segment/join/JoinableFactoryWrapper.java

```diff
@@ -340,12 +192,60 @@ static JoinClauseToFilterConversion convertJoinToFilter(
     return new JoinClauseToFilterConversion(null, false);
   }
 
+  public JoinableFactory getJoinableFactory()
+  {
+    return joinableFactory;
+  }
+
+  /**
+   * Compute a cache key prefix for a join data source. This includes the data sources that participate in the RHS of a
+   * join as well as any query specific constructs associated with join data source such as base table filter. This key prefix
+   * can be used in segment level cache or result level cache. The function can return following wrapped in an
```

Comment:
A bunch of this code looks like it changed, but I think it just had methods re-ordered?  Was that intentional?

Reply:
The IDE did this. I am using the druid intellij code formatter. Let me check how this can be avoided